ui/public/index.html

```diff
@@ -2,7 +2,27 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link name="unprocessed-script" content="lockdown.umd.js" />
+    <script src="./lockdown.umd.js"></script>
```

Comment:
`./` is superfluous here, makes it look like a module specifier, though it’s a relative URL.

Reply:
Removed